erpnext/support/doctype/issue/issue.py

```diff
@@ -391,3 +398,32 @@ def get_holidays(holiday_list_name):
 	holiday_list = frappe.get_cached_doc("Holiday List", holiday_list_name)
 	holidays = [holiday.holiday_date for holiday in holiday_list.holidays]
 	return holidays
+
+
+@frappe.whitelist()
+def create_new_task_from_issue(self, assign_to):
+	self = frappe.json.loads(self)
+	doc = frappe.new_doc("Task")
+	doc.subject = self.get("subject")
+	doc.issue = self.get("name")
+	doc.description = self.get("description")
+	doc.priority = self.get("priority")
+	doc.status = self.get("status")
+	doc.project = self.get("project")
+	doc.save()
+	args = {
+		"assign_to": [assign_to],
+		"doctype": "Task",
+		"name": doc.name,
+	}
+	add(args)
+	frappe.msgprint("Task Created Successfully , {0}".format(get_link_to_form("Task", doc.name)))
```

Comment:
https://frappeframework.com/docs/user/en/translations 
```suggestion
	frappe.msgprint(_("Task Created Successfully: {0}").format(get_link_to_form("Task", doc.name)))
```